src/main/java/com/regalii/regaliator/api/Response.java

```diff
@@ -74,7 +74,9 @@ public void printHeaders() {
     public String getXPaginationHeader() {
       Map<String, List<String>> hdrs = connection.getHeaderFields();
 
-      return (String) hdrs.get("X-Pagination").toArray()[0];
+      // Response headers should be case-insensitive, and VGS as a proxy will
+      // coerce headers to lowercase in compatibility with HTTP/2.
```

Comment:
As discussed, VGS proxy does not actually coerce headers to be lowercase, it actually just preserves the headers as they are. It is only when VGS receives a request wherein the headers were already lowercase that Arcus would see lowercase headers.

Reply:
Okay that's fair. Will update the comment to say it's more of an HTTP2 thing that should be handled by the SDK